Incomplete Makefile

Thanks for the excellent article at https://engineering.docker.com/2019/02/addressing-time-drift-in-docker-desktop-for-mac/ that pointed me to this repo.

I'm trying to use the top level Makefile, and ran into some issues.
1)
```make run
ChaimGs-MacBook-Air:hyperkit-measure-time-drift cgeretz$ make
make: *** No rule to make target `cmd/client/client', needed by `binaries'.  Stop.
```
2\)
Also current Makefile won't build the server image either, though the below diff seems to take care of that problem.

```diff
diff --git a/Makefile b/Makefile
index 6f5b834..f5dab49 100644
--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,7 @@ drift.0.postprocessed.dat: drift.0.dat
 binaries: server cmd/client/client
 
 .PHONY: server
+server:
        (cd cmd/server && docker build -t server .)
 
 .PHONY: clean
```

--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,11 @@ drift.0.postprocessed.dat: drift.0.dat
 
 binaries: server cmd/client/client
 
+cmd/client/client:
+	cd cmd/client && go build
+
 .PHONY: server
+server:
 	(cd cmd/server && docker build -t server .)
 
 .PHONY: clean

